.github/actions/repo-gardening/src/tasks/check-description/index.js

```diff
@@ -0,0 +1,212 @@
+/**
+ * External dependencies
+ */
+const moment = require( 'moment' );
+
+/**
+ * Internal dependencies
+ */
+const debug = require( '../../debug' );
+const getLabels = require( '../../get-labels' );
+const getNextValidMilestone = require( '../../get-next-valid-milestone' );
+
+/* global GitHub, WebhookPayloadPullRequest */
+
+/**
+ * Check if a PR has unverified commits.
+ *
+ * @param {GitHub} octokit - Initialized Octokit REST client.
+ * @param {string} owner   - Repository owner.
+ * @param {string} repo    - Repository name.
+ * @param {string} number  - PR number.
+ *
+ * @returns {Promise<boolean>} Promise resolving to boolean.
+ */
+async function hasUnverifiedCommit( octokit, owner, repo, number ) {
+	for await ( const response of octokit.paginate.iterator( octokit.pulls.listCommits, {
+		owner: owner.login,
+		repo,
+		pull_number: +number,
+	} ) ) {
+		response.data.map( commit => {
+			if ( commit.commit.message.includes( '[not verified]' ) ) {
+				return true;
+			}
+		} );
+	}
+
+	return false;
+}
+
+/**
+ * Check for status labels on a PR.
+ *
+ * @param {GitHub} octokit - Initialized Octokit REST client.
+ * @param {string} owner   - Repository owner.
+ * @param {string} repo    - Repository name.
+ * @param {string} number  - PR number.
+ *
+ * @returns {Promise<boolean>} Promise resolving to boolean.
+ */
+async function hasStatusLabels( octokit, owner, repo, number ) {
+	const labels = await getLabels( octokit, owner, repo, number );
+
+	// We're really only interested in status labels
+	labels.map( label => {
+		if ( label.includes( '[Status]' ) ) {
+			debug( `check-description: this label (${ label }) includes the word Status` );
+			return true;
```

Comment:
>  Figure out why label detection doesn't work properly.

Maybe because this returns from the map function, not `hasStatusLabels`. `hasStatusLabels` will always return false at line 62.

Looks like `hasUnverifiedCommit` above has the same bug.

Reply:
Ah yes, you're right. This should be better as of 23672cc00f252a4e527a480982a7f1c176a9d191